### DISCUSSION TITLE
what are we suppose to do with Section 3 NFT?

### DISCUSSION THREAD

**Discussion by Vincent71399** [Cyfrin/security-and-auditing-full-course-s23#208] [Q&A] [answered]
Should I take the Section 3 NFT contracts as an auditing exercise?

**Comment by EngrPips** [comment, marked as answer]
Hello @Vincent71399, If there is an NFT challenge for a section you are supposed to solve the challenge by minting the NFT to your address to prove that you understood properly everything that was taught in the section.

**Reply by Vincent71399** [reply]
I see, so it is a puzzle like Ethernaut, rather than an audit exercise right?

**Reply by EngrPips** [reply]
Yeah, you can look at it like that: an exercise that, if you successfully solve it, you will get an NFT as proof of success.
